.readthedocs.yml

```diff
@@ -5,7 +5,7 @@
 version: 2
 formats: []
 build:
-  image: latest
+  os: ubuntu-22.04
```

Comment:
@dmick see https://docs.readthedocs.io/en/stable/config-file/v2.html#build-os . so we only have two options: ubuntu-20.04, ubuntu-22.04

probably we could also have referenced it in the commit message, and explain that "ubuntu-22.04" is preferred, as (i guess), presumably, it is newer and hence better, like with newer tool chain and is better supported, etc.

Reply:
@dmick there was some discussion about adding "latest"/"ubuntu-latest-lts" as an option in https://github.com/readthedocs/readthedocs.org/issues/8861
but seems like they decided against it because they wanted to avoid breaking users builds whenever they updated "latest" so they're letting users update on their own timeline. Their explanation:  https://github.com/readthedocs/readthedocs.org/issues/10401#issuecomment-1581020801

@tchaikov I'll make sure to mention the reason of using a version more explicitly whenever there's next update